script.js

```diff
@@ -26,4 +27,175 @@ document.querySelectorAll('.mobile-menu-list-item').forEach((link) => {
     }, 900);
     hamburgerButton.style.display = 'block';
   });
-});
\ No newline at end of file
+});
+
+/* ==========Details Popup Window========== */
+// Create project object
+const projectPop = [
```

Comment:
- [ ] You should implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- You should implement popups for both mobile and desktop screen sizes:
> **_Mobile popup_**

- [ ]  In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.

- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to the source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.

> __ I've checked on your project's live version and every time I click on the pop-up window for each `see this project`  button, it doesn't show me any pop-up window, neither in the desktop or mobile version, so I suggest you to kindly check on it, also when you do that, make sure you implement the links of source code (your GitHub link to the project) and the live version of each project you feature if you don't have any project to showcase there just put a link `"/"` make sure no links are broken, if you don't have a link to redirect just use this `"/"`.

Make sure to work on these changes so your project can be soon approved, good luck on everything 👍😊
![giphy (1)](https://user-images.githubusercontent.com/54015740/151432042-9ce8ee13-dfe4-48d0-b48f-43cf221f4de6.gif)